client/src/components/surveys/SurveyNew.js

```diff
@@ -0,0 +1,34 @@
+// SurveyNew shows the SurveyForm and SurveyFormPreview
+import React, { Component } from 'react';
+import { reduxForm } from 'redux-form';
+import SurveyForm from './SurveyForm';
+import SurveyFormReview from './SurveyFormReview';
+
+class SurveyNew extends Component {
+  //state initialization
+  state = { showFormReview: false };
+
+  renderContent() {
+    if (this.state.showFormReview) {
+      return (
+        <SurveyFormReview
+          onCancel={() => this.setState({ showFormReview: false })}
```

Comment:
Don't need the above if statement should be able to do `<SurveyFormReview onCancel={() => this.setState({ showFormReview: !this.state.showFormReview })}`